src/main/resources/se/diabol/jenkins/pipeline/DeliveryPipelineView/configure-entries.jelly

```diff
@@ -27,6 +27,10 @@
             <f:select/>
         </f:entry>
 
+        <f:entry title="Max number of pipelines" field="maxNumberOfJobs" default="-1">
+            <f:textbox/>
+        </f:entry>
```

Comment:
Let's go a head with the current solution with the below code comments addressed. I think the proposed solution is sufficient for this use case.